src/adaptors/tcp_adaptor.c

```diff
@@ -677,6 +678,19 @@ static void qdr_tcp_connection_ingress_accept(qdr_tcp_connection_t* tc)
     allocate_tcp_buffer(&tc->read_buffer);
     tc->remote_address = get_address_string(tc->pn_raw_conn);
     tc->global_id = get_global_id(tc->config.site_id, tc->remote_address);
+
+    // Create the tcp connection properties map.
+    pn_data_t *props = pn_data(0);
```

Comment:
Make this a static function and call it from both places ?

Reply:
Ok. Eventually consoles and such will want to differentiate between infrastructure connections (egress_dispatch) and user connections that consume router resources by sending/receiving traffic. Making them all the same is good enough for now.